### ISSUE TITLE
Which release version should I use?

### ISSUE THREAD

**Issue by Imzxhir** [utsanjan/PD-Runner#56]
Mac Model: iMac (27-inch, Mid 2010)
macOS Version: High Sierra 10.13.6

**Comment by CheemsC** [comment]
You should use the latest version if you have the latest version of Parallels Desktop.

**Comment by CheemsC** [comment]
No, PD Runner v18.1.0 is only for the latest version of Parallels Desktop, which is 18.1.0. I would try using some of the older versions of PD Runner, or update your Parallels to this latest version.

**Comment by CheemsC** [comment]
Yes. Just keep trying different versions until it works. If you get errors, check the readme.

**Comment by CheemsC** [comment]
Your only option would be to update your Mac, if you are able to, or look for a different tool/way to use Parallels for free.

**Comment by utsanjan** [comment]
> Mac Model: iMac (27-inch, Mid 2010) macOS Version: High Sierra 10.13.6

**Comment by utsanjan** [comment]
> Mac Model: iMac (27-inch, Mid 2010) macOS Version: High Sierra 10.13.6

You can use the PD Runner v0.3.8 from the releases, and then follow the below-given steps:
- Extract the zip file and get the "PD Runner.app"
- Copy "PD Runner.app" to the "Applications" Folder
- Run the following command in the terminal with admin rights
```
sudo chmod +x /Applications/PD\ Runner.app/patch && sudo sh /Applications/PD\ Runner.app/patch
```

Finally, open parallels directly from the app drawer, and you're good to go.

**Comment by utsanjan** [comment]
> It keeps saying: ![image](https://user-images.githubusercontent.com/98621617/202258407-dca623bb-7900-4bda-8c66-e8fc1c3ea86c.png)

Have you moved the PD Runner.app to the applications folder?

**Comment by utsanjan** [comment]
Please try downloading it once more from here https://github.com/utsanjan/PD-Runner/releases/tag/0.3.8
Then repeat the process. Let me know what happens next.

**Comment by utsanjan** [comment]
Did it say "Patching completed successfully" in the terminal?

**Comment by utsanjan** [comment]
Try to restart the pc once then open parallels

**Comment by utsanjan** [comment]
> > Most probably it will take more time to open the OS, I'm not sure if your PC is a bit slow, is it?
> 
> My Mac is fast

Then maybe it is some version mismatch going on here. The parallels version is not matching with the version of the patched parallels license. You should try updating your macOS.

**Comment by CheemsC** [comment]
You can try using an app that allows you to update your mac even though it is not supported. Search this up on YouTube, there are a lot of videos about this. There is also a github page for one of these tools, although I don't remember its name off the top of my head.

**Comment by CheemsC** [comment]
I see. I recommend just going with the method of changing the time back before every time you launch parallels, and then setting it back.

**Comment by CheemsC** [comment]
I'm sorry, but I don't know what to tell you, dude. I would keep searching for a method, or perhaps turn to torrents. I have seen plenty of safe Parallels Desktop cracks with many comments and good ratings. I would look in that direction if you are sure you know what you are doing. Let me know how it goes.

**Comment by CheemsC** [comment]
https://www.mediafire.com/folder/sifi4f8h0d86m/Parallels_Desktop
This is a link to the whole repo of Parallels Desktop Cracks from iCrackMac.

**Comment by CheemsC** [comment]
I would try installing 17.1.4 with the existing app already installed, and if that doesn't work, uninstall Parallels and install the version you want. When you uninstall PD, your VMs should not be deleted.

**Comment by CheemsC** [comment]
<img width="759" alt="image" src="https://user-images.githubusercontent.com/118298206/202298259-f8fd3e85-ea15-43a8-82de-eae18134d999.png">

**Comment by CheemsC** [comment]
"Sign out of your Parallels account, if you have one, and completely close the Parallels Desktop program."

**Comment by CheemsC** [comment]
> 

Yes. I fail to see how this has anything to do with the original installer. The only installer you should be opening is the one that comes included in the folder you download...

**Comment by CheemsC** [comment]
All this first step is telling you is to sign out of your account and quit the Parallels app. The steps after this detail what you need to do with the cracked files you download.
Here is a video if you require further reference, although I'm not sure if it is the same process...:
https://www.youtube.com/watch?v=OlIQobzYZP8

**Comment by CheemsC** [comment]
Someone was recommending this video on a torrent website, which it probably the same crack on iCrackMac...

**Comment by CheemsC** [comment]
Let me know how it goes. I am also having issues with PD Runner aas you might see in one of the issues I made. For me, the time changing trick seems to work, but I might go the iCrackMac route very soon.

**Comment by CheemsC** [comment]
Did you manage to fix it?

**Comment by CheemsC** [comment]
I'd look at some of these cracks: https://1337x.to/sort-search/Parallels%20Desktop/time/desc/1/
Use a VPN, though.

**Comment by CheemsC** [comment]
https://pirate-bays.net/search?q=parallels+desktop

**Comment by CheemsC** [comment]
I saw a 17.1.4 there

**Comment by CheemsC** [comment]
Patched refers to the same thing as "Patch" in PD Runner.

**Comment by CheemsC** [comment]
I'd try uninstalling and reinstalling parallels, and then retrying PD Runner, changing the time back, and iCrackMac.
(Also, the crack on iCrackMac is very likely the same one on 1337x and Pirate Bay.)

**Comment by CheemsC** [comment]
And if that doesn't work, try other Virtualizing solutions or...
https://www.parallels.com/products/desktop/buy/

**Comment by CheemsC** [comment]
UTM, VMWare Fusion, Virtual Box...
Also, its _only_ $99... haha



_(overpriced)_

**Comment by CheemsC** [comment]
https://download.parallels.com/desktop/v17/17.1.4-51567/ParallelsDesktop-17.1.4-51567.dmg

**Comment by CheemsC** [comment]
I guess you can't use PD Runner 🤷‍♂️
